common/utils/src/main/resources/error/error-conditions.json

```
@@ -5315,6 +5315,11 @@
         "message" : [
           "Update column nullability for MySQL and MS SQL Server."
         ]
+      },
+      "WRITE_FOR_BINARY_SOURCE" : {
+        "message" : [
+          "Write is not supported for binary file data source."
```

"is not supported" can be removed. See the full error message:
```
The feature is not supported: Write is not supported for binary file data source.
```
How about:
```suggestion
          "Write for the binary file data source."
```

---

Makes sense. Thanks for the suggestion!